tests/unit/fem/test_calcshape.cpp

```diff
@@ -173,7 +173,7 @@ TEST_CASE("CalcShape for several H1 FiniteElement instances",
          std::cout << "Testing H1_SegmentElement::CalcShape() "
                    << "for order " << order << std::endl;
          H1_SegmentElement fe(order);
-         TestCalcShape(&fe, resolution);
+         TestCalcShape(&fe, resolution, 2e-11*std::pow(10, order));
```

Comment:
I'd expect a milder deterioration of the accuracy with the order. Is it really that bad?

Reply:
I don't know if the asymptotic scaling is right, but it was a pretty good fit for the numbers for orders 1 through 5. I even had to adjust the constant up because of runs on lassen.

Reply:
This shouldn't stop this PR, but it could be indeed interesting to understand why this operation is so badly conditioned. I also did not expect it.

Reply:
It looks like the test is evaluating basis functions outside the reference domian -- I'm not sure why this is done -- and that probably explains the bad behavior.

Reply:
Oh, good observation